docs/backend.md

```diff
@@ -111,6 +111,25 @@ def bar(run, value: "var#foo"):
     return value * 2
 ```
 
+Pattern matching using unix shell-style
+[wildcards](https://docs.python.org/3/library/fnmatch.html) is allowed in `var#`
+annotations. In such case, a `dict` is returned where keys are the name of
+matching dependencies and values are their results:
+```python
+@Variable()
+def base_1(run):
+    return 1
+
+@Variable()
+def base_2(run):
+    return 2
+
+@Variable()
+def sum_values(run, data: 'var#base_?'):
```

Comment:
Small nitpick: I might illustrate the wildcards with `*`, because I think more people will recognise that as a familiar mechanism. I know what `?` means, but it's much rarer, so I assume a lot of people don't know it's part of the same system.

Reply:
updated in 847df374ee47f675fc6a70b8ed26beea4658badb